changelog/22468.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core/metrics: vault.raft_storage.bolt.write.time should be a counter not a summary
```

Comment:
We should probably also delete the Changelog file for the previous PR right? Otherwise both will show up in the next release which is confusing!

Reply:
Yes, thank you again. Apparently my changelog-fu needs some work 😞